core/modules/qdisp/ExecStatus.cc

```diff
@@ -53,6 +53,8 @@ char const* ExecStatus::stateText(ExecStatus::State s) {
     case RESPONSE_ERROR: return "Response error";
     case RESPONSE_DATA: return "Retrieving response data";
     case RESPONSE_DATA_ERROR: return "Error retrieving response";
+    case RESPONSE_DATA_ERROR_OK: return "Error retrieving response session OK";
```

Comment:
I can't say that I fully understand what is happening with this particular error. What it means to say is that there was an error or something unexpected in the response received from xrootd, but the xrootd session is still OK. The RESPONSE_DATA_ERROR_CORRUPT means that there was an error in the xrootd response and there is something wrong with the xrootd session. Really understanding this requires more xrootd knowledge than I currently have.